### PR TITLE
Showing FunctionQuery in explain

### PR DESCRIPTION
**Pull request by ThoWagen**
This PR adds support for "FunctionQuery" in explain.

### CODE REVIEW

File: languages/de.ini [hunk 1]
```diff
@@ -549,6 +549,7 @@ From = "Von"
 Full description = "Ausführliche Beschreibung"
 Full text is not displayed to guests = "Volltext ist per Gastzugang nicht verfügbar."
 fulltext_limit = "Eingrenzen auf Datensätze mit Volltextzugriff"
+Function = "Funktion"
```
Comment (demiankatz) [inline languages/de.ini]: Maybe we should use a more descriptive key like `explain_function_query_label` to help the translators understand the context here. (I realize this feedback might have been valid for some of the other labels used by explain, but this word is particularly ambiguous, so clarifying it seems especially important to ensure the text is not inappropriately reused in the future).

**Review by demiankatz — CHANGES_REQUESTED** [review]
Thanks, @ThoWagen, this works well for me -- just one suggestion regarding the i18n.

File: languages/de.ini (hunk 1 above)
Comment (ThoWagen) [reply languages/de.ini]: Yes that makes sense. I changed it.

**Review by demiankatz — APPROVED** [review]
Thanks, @ThoWagen!

**Pull request merged**